saplings/circuits/src/components/NodeCard.scss

```diff
@@ -121,3 +112,10 @@
     }
   }
 }
+
+.node-card.selectable {
+  cursor: pointer;
+}
+.node-card.selectable:hover {
+  background-color: var(--color-light-grey);
```

Comment:
This as a background color here doesn't have enough contrast with the text color on the metadata chip. I would either make this lighter or make the text of the metadata chip darker so that we work towards fully accessible colors.